psydac/linalg/tests/test_solvers.py

```diff
@@ -207,6 +227,189 @@ def test_solver_tridiagonal(n, p, dtype, solver, verbose=False):
         assert errh_norm < tol
         assert solver == 'pcg' or errc_norm < tol
 
+#===============================================================================
+def test_LST_preconditioner(comm=None):
+
+    ncells_3d      = [16, 7, 11]
+    degree_3d      = [1, 4, 2]
+    periodic_3d    = [False, True, False]
```

Comment:
We can remove some whitespace here
```suggestion
    ncells_3d   = [16, 7, 11]
    degree_3d   = [1, 4, 2]
    periodic_3d = [False, True, False]
```